### PR TITLE
Translated the "Options" window

### PR DESCRIPTION
**Pull request by gdabi**
All the strings found on the "Options" window have been translated into spanish

### CODE REVIEW

File: data/language/spanish_sp.txt [hunk 1]
```diff
@@ -3456,9 +3456,521 @@ STR_3444    :Page 5
 STR_3445    :Set Patrol Area
 STR_3446    :Cancelar Patrol Area
 
-#Thousands separator
-STR_5151    :.
-#Decimal separator
-STR_5152    :,
```
Comment (Gymnasiast) [inline data/language/spanish_sp.txt]: What does Spanish use to separate decimals and what to separate thousands?

Comment (gdabi) [inline data/language/spanish_sp.txt]: In fact, we use the dot to separate thousands, and comma to separate decimals

My mistake. I replaced these four lines by all the new STR_5XXX strings and I forgot to check that the new STR_5152 and STR_5152 were correct

File: data/language/spanish_sp.txt [hunk 2]
```diff
@@ -3456,9 +3456,521 @@ STR_3444    :Page 5
...
+STR_5142    :Normal Speed
+STR_5143    :Quick Speed
+STR_5144    :Fast Speed
+STR_5145    :Turbo Speed
+STR_5146    :Hyper Speed
+STR_5147    :Trucos
+STR_5148    :{SMALLFONT}{BLACK}Change the game speed
+STR_5149    :{SMALLFONT}{BLACK}Show cheat options
+STR_5150    :Activar herramientas de depuración
 STR_5151    :.
```
Comment (gdabi) [inline data/language/spanish_sp.txt]: @Gymnasiast Now the separators are correct. Thanks

File: data/language/spanish_sp.txt [hunk 3]
```diff
@@ -3456,9 +3456,521 @@ STR_3444    :Page 5
 STR_3445    :Set Patrol Area
 STR_3446    :Cancelar Patrol Area
 
-#Thousands separator
+# New strings, cleaner
```
Comment (Gymnasiast) [inline data/language/spanish_sp.txt]: You can leave out this line, it's a note for UK English.

**Comment by Gymnasiast** [comment]
Thanks!

**Pull request merged**